src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/oracle/OracleEngineSchemaTest.java

```diff
@@ -97,10 +112,101 @@ public void testSystemGeneratedColumns() throws Exception {
                     .build();
             engine.addEntity(entity);
 
-            assertFalse("The simulated system generated column should not appear in the table metadata", engine.getMetadata("TEST_SYS_COL").containsKey("SYS_COL1"));
-            assertTrue("The regular column should appear in the table metadata", engine.getMetadata("TEST_SYS_COL").containsKey("COL1"));
-        } finally {
-            engine.close();
+            assertFalse(
+                    "The simulated system generated column should not appear in the table metadata",
+                    engine.getMetadata("TEST_SYS_COL").containsKey("SYS_COL1")
+            );
+            assertTrue(
+                    "The regular column should appear in the table metadata",
+                    engine.getMetadata("TEST_SYS_COL").containsKey("COL1")
+            );
         }
     }
+
+    /**
+     * This method tests that, when the {@link PdbProperties pdb.compress_lobs} is true (which is the default value),
+     * the LOBS columns are compressed: Both BLOB and CLOB types are tested.
+     *
+     * @throws Exception if anything goes wrong with the test
+     */
+    @Test
+    public void testCompressLobs() throws Exception {
+        final String tablespace = "TEST_TABLESPACE";
+        final String tableName = "TEST_TABLE";
+        final String tableUserLobs = "USER_LOBS";
+
+        final String tableId = "ID";
```

Comment:
maybe these 3 should be called _column*_ or _*column_
instead of _table*_

Reply:
Yup, makes sense